python/docs/source/conf.py

```diff
@@ -0,0 +1,69 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+from typing import List
+from recommonmark.transform import AutoStructify
+
+# Set python project root
+sys.path.insert(0, os.path.abspath("../../"))
+
+# The master toctree document
+master_doc = "index"
+
+# -- Project information -----------------------------------------------------
+
+project = "ml4ir"
+copyright = "2020, Search Relevance (Salesforce.com, Inc.)"
+author = "Search Relevance (Salesforce.com, Inc.)"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0"
```

Comment:
this should be `0.2.0`, right ?

Reply:
Ah. Yes. I was trying to find this. Thank you :)

Reply:
Should we also release to pypi?

Reply:
I have already released latest version to pypi @balikasg